internal/compress/gob_test.go

```diff
@@ -92,31 +54,32 @@ func TestNewGob(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns (Compressor, nil) when option is empty",
```

Comment:
please create `returns (Compressor, nil) when option is not empty` test case :)

Reply:
sorry. I already created this one.
https://github.com/vdaas/vald/blob/test/internal/compress/gob/internal/compress/gob_test.go#L72-L75

and gob has no option.